eng/ProjectReferences.props

```diff
@@ -107,6 +107,7 @@
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Mvc.Razor" ProjectPath="$(RepositoryRoot)src\Mvc\Mvc.Razor\src\Microsoft.AspNetCore.Mvc.Razor.csproj" RefProjectPath="$(RepositoryRoot)src\Mvc\Mvc.Razor\ref\Microsoft.AspNetCore.Mvc.Razor.csproj" />
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Mvc.TagHelpers" ProjectPath="$(RepositoryRoot)src\Mvc\Mvc.TagHelpers\src\Microsoft.AspNetCore.Mvc.TagHelpers.csproj" RefProjectPath="$(RepositoryRoot)src\Mvc\Mvc.TagHelpers\ref\Microsoft.AspNetCore.Mvc.TagHelpers.csproj" />
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Mvc.Testing" ProjectPath="$(RepositoryRoot)src\Mvc\Mvc.Testing\src\Microsoft.AspNetCore.Mvc.Testing.csproj" RefProjectPath="$(RepositoryRoot)src\Mvc\Mvc.Testing\ref\Microsoft.AspNetCore.Mvc.Testing.csproj" />
+    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Mvc.ViewFeatures.Prerendering" ProjectPath="$(RepositoryRoot)src\Mvc\Mvc.ViewFeatures.Prerendering\src\Microsoft.AspNetCore.Mvc.ViewFeatures.Prerendering.csproj" RefProjectPath="$(RepositoryRoot)src\Mvc\Mvc.ViewFeatures.Prerendering\ref\Microsoft.AspNetCore.Mvc.ViewFeatures.Prerendering.csproj" />
```

Comment:
Can we drop ViewFeatures from the name? ViewFeatures is named like it is because it's a pile of unorganized things. I don't want to spread the terminology contagion.

Reply:
Mvc.Components?